pitest/src/test/java/org/pitest/coverage/execute/DependencyFilterTest.java

```diff
@@ -64,13 +65,13 @@ public void shouldNotPerformAnalysisWhenDependencyDistanceIsLessThan0()
   public void shouldReturnOnlyTestUnitsForClassesWithinReach()
       throws IOException {
     when(
-        this.extractor.extractCallDependenciesForPackages(eq(this.aTestUnit
-            .getDescription().getFirstTestClass()), any(Predicate.class)))
-            .thenReturn(Arrays.asList("foo"));
+        this.extractor.extractCallDependenciesForPackages(
+            eq(this.aTestUnit.getDescription().getFirstTestClass()),
+            isNull())).thenReturn(Arrays.asList("foo"));
     when(
         this.extractor.extractCallDependenciesForPackages(
             eq(this.anotherTestUnit.getDescription().getFirstTestClass()),
-            any(Predicate.class))).thenReturn(Collections.<String> emptyList());
+            isNull())).thenReturn(Collections.<String> emptyList());
```

Comment:
This null should never happen because it will lead to a NPE in DependencyExtractor.

I think it would be better to mock the Predicate<String> here as well (used as second constructor paramter).

Reply:
I've added always true predicate into `DependencyFilter` instead of `null`